src/Request/Admin/RequestHomebyAdmin.js

```diff
@@ -227,7 +227,7 @@ class RequestHomebyAdmin extends React.Component {
                         <input type="checkbox" />
                       </td>
                       <td>
-                        <Link to={`/Request/Admin/${request._id}`} style={{ color: 'black' }}>
+                        <Link to={`/Request/Admin/Answer/${request._id}`} style={{ color: 'black' }}>
```

Comment:
Thank you for catching this, I do have a component for request admin detail page but somehow it's not here... I'll fill an issue and fix this

Reply:
follow-up issue: #244